src/proxy/http/glue.rs

```diff
@@ -35,13 +35,13 @@ pub enum HttpBody {
 
 /// Glue for `tower_h2::Body`s to be used in hyper.
 #[derive(Debug, Default)]
-pub(super) struct BodyPayload<B> {
+pub struct BodyPayload<B> {
```

Comment:
Do these need to be `pub`, or can they just be `pub(in proxy::http)` or something?

Reply:
they can probably be `pub(in proxy)` (i wasn't aware of the _in_ syntax...)